packages/master-detail-layout/src/vaadin-master-detail-layout.d.ts

```diff
@@ -45,20 +45,21 @@ export interface MasterDetailLayoutEventMap extends HTMLElementEventMap, MasterD
  * `containment`  | Set to `layout` or `viewport` depending on the containment.
  * `orientation`  | Set to `horizontal` or `vertical` depending on the orientation.
  * `has-detail`   | Set when the detail content is provided.
- * `overlay`      | Set when the layout is using the overlay mode.
+ * `drawer`       | Set when the layout is using the drawer mode.
  * `stack`        | Set when the layout is using the stack mode.
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
- * @fires {CustomEvent} backdrop-click - Fired when the user clicks the backdrop in the overlay mode.
+ * @fires {CustomEvent} backdrop-click - Fired when the user clicks the backdrop in the drawer mode.
  * @fires {CustomEvent} detail-escape-press - Fired when the user presses Escape in the detail area.
  */
 declare class MasterDetailLayout extends SlotStylesMixin(ResizeMixin(ThemableMixin(ElementMixin(HTMLElement)))) {
   /**
    * Fixed size (in CSS length units) to be set on the detail area.
    * When specified, it prevents the detail area from growing or
    * shrinking. If there is not enough space to show master and detail
-   * areas next to each other, the layout switches to the overlay mode.
+   * areas next to each other, the layout switches to the overlay mode:
+   * either drawer or stack, depending on the `stackOverlay` property.
```

Comment:
```suggestion
   * shrinking. If there is not enough space to show master and detail
   * areas next to each other, the details are shown as an overlay:
   * either as drawer or stack, depending on the `stackOverlay` property.
```
We could make it more explicit what is shown in the overlay.

Reply:
Thanks, done.